core/unit_test/cuda/TestCuda_Spaces.cpp

```diff
@@ -254,6 +254,12 @@ TEST(cuda, space_access) {
           Kokkos::Impl::HostMirror<Kokkos::CudaHostPinnedSpace>::Space,
           Kokkos::HostSpace>::accessible,
       "");
+#if KOKKOS_ENABLE_CUDA_EVM
```

Comment:
Shouldn't this be `KOKKOS_ENABLE_CUDA_UVM`? This is something we need to be more careful about.  Make this `#ifdef` also, please, for consistency.

Related: we need a pattern of some sort to prevent these sort of typos in the future. One option for this is how the `KOKKOS_IMPL_ENABLE_DEVICE_MULTIVERSIONING` and `KOKKOS_IMPL_DISABLE_DEVICE_MULTIVERSIONING` macros in my `Kokkos::get` pull request #2407

Reply:
Fixed.